addon/ng2/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -1,24 +1,23 @@
 {
-  "compileOnSave": false,
   "compilerOptions": {
+    "baseUrl":"./src",
     "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "mapRoot": "/",
```

Comment:
That didn't change, it's like that in master. I think there's a few things we should remove from the current tsconfig, it's a bit bloated.